Config/LocustKass_FreeSpace_Template.xml.in

```diff
@@ -1,66 +1,29 @@
-
 <!-- parameters -->
-
-<external_define name="fieldX" value="0.0"/>
-<external_define name="fieldY" value="0.0"/>
-
- <external_define name="generator" value="gen_uniform"/>
+<external_define name="simulation" value="1"/>
```

Comment:
This line can be deleted. The variable is never used.

Reply:
The line is now deleted.  Thanks.